code-challenges.js

```diff
@@ -47,3 +143,32 @@ const number2 = 100
 // Set two:
 const number3 = 27
 const number4 = 24
+
+//make a function called subtractor
+//the function checks (makes a decision -> conditional) which input (argument) num1 , num2  is bigger than the other.
+//then the function subtracts the larger number from the smaller number.
+//the output is then invoked using console.log();because it is a function
+//what if both numbers are the same value?
+
+const subtractor = (num1 , num2) => {
+  if(num1 > num2){
+    return `${num1 - num2}`
+  }else if(num2 > num1){
+    return `${num2 - num1}`
+  }else{
+    return `The numbers are the same value, but lets carry on. ${num1 - num2}`
+  }
+}
+console.log(subtractor(number1 , number2));
+console.log(subtractor(number3 , number4));
+console.log(subtractor(number1 , number3));
+console.log(subtractor(number2 , number4));
+console.log(subtractor(42 , -42));
+console.log(subtractor(3 , 3));
+//my outputs:
+// 42
+// 3
+// 31
+// 76
+// 84
+// The numbers are the same value, but lets carry on. 0
```

Comment:
Line 159 is a crack up. I feel like its a zero that will always have a lot of meaning. Great work!